<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Driver_Profile</class>
 <widget class="QWidget" name="Driver_Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Driver Profile</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Driver_FName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Driver_LName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Driver_Email">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Driver_PNumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Update_FName">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Update_LName">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Update_Email">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Update_PNumber">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Driver_NINumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Driver_DLNumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Update_NINumber">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Update_DLNumber">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Return_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>National Insurance Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Driving License Number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
